src/tasks/command/command_parser.c

```diff
@@ -32,8 +32,10 @@ void dispatch_command(slate_t *slate, packet_t *packet)
         case PAYLOAD_EXEC:
         {
             PAYLOAD_COMMAND_DATA payload_command;
-            strlcpy(payload_command.serialized_command, command_payload,

```

Comment:
My understanding is that strlcpy is able to add a null-terminator at the end of the string and performs truncation when necessary: https://linux.die.net/man/3/strlcpy